src/pages/en/blog/2023-05-19-qm-needs-you.md

```diff
@@ -0,0 +1,36 @@
+
+![A screenshot of a graph generated by Enigma to show unmapped entries, highlighting the world, entity and client packages are problem areas](/assets/img/writing/blog/2023-05-19-qm-needs-you/2023-05-19-mapping-stats.png)
+
+If mapping the game is less your thing, we have plenty of other projects for you to help out with. We can also use the help of those experienced with Java, networking, Swing, Gradle, and even translation.
+Contributors have a few great options:
+1. Working on our new toolchain, which we’re calling Quilt Mapping Tools. You can help with designing the toolchain, writing Java code, designing the new mapping format, or even preparing other parts of Quilt to use QMT. The best way to find out what needs doing here is talking to our team leader, OroArmor, on [our Discord](https://discord.quiltmc.org)! You can also check the [QMT issue tracker](https://github.com/QuiltMC/quilt-mapping-tools/issues) to get a rough idea of the work that’s still in progress.
+2. Helping out with our deobfuscator, Enigma! Improving Enigma helps us make mappings faster and with fewer mistakes. To help out with Enigma, you can look at *its* [issue tracker](https://github.com/QuiltMC/enigma/issues), or try to break it to help us find more bugs. Enigma is quite a complicated program, and you can always talk to me (rai), or Martrixx on [our Discord](https://discord.quiltmc.org) if you’d like some help understanding it. Enigma is separated into four parts: a backend, a command line interface, a server, and a swing GUI, which means there’s something for every type of Java experience! If the language you speak isn’t English *or* Java, Enigma also has internationalisation support! You can add languages in the [lang](https://github.com/QuiltMC/enigma/tree/master/enigma/src/main/resources/lang) folder, or complete translation for existing languages! Compare against the `en_us` file to find missing translations.
```

Comment:
Might as well add a notice for our favorite `help wanted` issue, the rare (or sometimes not) QuiltMC/enigma#108